setup.py

```diff
@@ -50,7 +50,7 @@
         ]
     },
     python_requires=">=3.7.0",
-    install_requires=["numpy>=1.17", "packaging>=20.0", "psutil", "pyyaml", "torch>=1.4.0"],
+    install_requires=["numpy>=1.17", "packaging>=20.0", "psutil", "pyyaml", "torch>=1.4.0", "rich"],
```

Comment:
You put `rich` in the core install here but use an `is_rich_available` later on.

Reply:
As discussed offline, opted to make rich a true dep in Accelerate